cartoframes/auth/context.py

```diff
@@ -228,7 +228,7 @@ def read(self, table_name, limit=None, decode_geom=False, shared_user=None, retr
         # choose schema (default user - org or standalone - or shared)
         schema = 'public' if not self.is_org else (shared_user or self.creds.username)
 
-        dataset = Dataset(table_name, schema=schema, credentials=self)
+        dataset = Dataset(table_name, schema=schema, credentials=self.creds)
```

Comment:
Minor thing: I think we should use `credentials` instead of `creds` as we're doing in the rest of the classes.

Reply:
We are going to remove this file in the next PR (this is the old context). That's why there are no naming updates here.

Reply:
I see! ok :)